src/mainboard/opentitan/crb/src/main.rs

```diff
@@ -0,0 +1,97 @@
+#![feature(asm)]
+#![feature(lang_items, start)]
+#![no_std]
+#![no_main]
+#![feature(global_asm)]
+#![deny(warnings)]
+
+use core::intrinsics::transmute;
+use core::panic::PanicInfo;
+use core::sync::atomic::{AtomicUsize, Ordering, spin_loop_hint};
+use core::fmt::Write;
+use model::Driver;
+use payloads::payload;
+use print;
+use uart::sifive::SiFive;
```

Comment:
nope

Reply:
https://github.com/lowRISC/opentitan/blob/master/sw/device/lib/uart.c to see how it works. Wow, the opentitan sw tree is .. hmm.

Reply:
This is a placeholder until we get a opentitan driver written. Is there a spec for it?